packages/sanity/src/core/comments/plugin/input/helpers.ts

```diff
@@ -107,10 +107,9 @@ export function useAuthoringReferenceElement(
 export function getSelectionBoundingRect(): DOMRect | null {
   const selection = window.getSelection()
   let range = null
-  try {
+
+  if (selection && selection.rangeCount > 0) {
     range = selection?.getRangeAt(0)
```

Comment:
NIT, selection will always be defined here because of the previous check.
```suggestion
    range = selection.getRangeAt(0)
```

Reply:
Good catch :) changed 👍